### PR TITLE
Re-enable trace_transaction and trace_get - issue 1833

### PR DESCRIPTION
**Pull request by tjayrush**

### CODE REVIEW

File: cmd/rpcdaemon/commands/trace_filtering.go [hunk 1]
```diff
@@ -23,19 +23,73 @@ import (
...
-// TODO(tjayrush): I think this should return an []interface{}, so we can return both Parity and Geth traces
 func (api *TraceAPIImpl) Transaction(ctx context.Context, txHash common.Hash) (ParityTraces, error) {
 	tx, err := api.kv.BeginRo(ctx)
 	if err != nil {
 		return nil, err
 	}
 	defer tx.Rollback()
 
-	traces, err := api.getTransactionTraces(tx, ctx, txHash)
+	txn, _, blockNumber, _ := rawdb.ReadTransaction(ethdb.NewRoTxDb(tx), txHash)
```
Comment (AlexeyAkhunov) [inline cmd/rpcdaemon/commands/trace_filtering.go:33]: I think you should also keep the 4th return value (`txIndex`) and use it later to only execute until that `txIndex` later

File: cmd/rpcdaemon/commands/trace_filtering.go [hunk 2]
```diff
@@ -23,19 +23,73 @@ import (
...
+			})
+		}
+	}
+
+	baseBn := bn
+	if baseBn > 0 {
+		baseBn -= 1
+	}
+
+	traces, err := api.callManyTransactions(ctx, tx, txs, hash, rpc.BlockNumber(baseBn))
```
Comment (AlexeyAkhunov) [inline cmd/rpcdaemon/commands/trace_filtering.go:73]: This only needs to execute transactions `0..txIndex` and then we only need to take the latest trace

File: cmd/rpcdaemon/commands/trace_filtering.go (hunk 1 above)
Comment (tjayrush) [reply cmd/rpcdaemon/commands/trace_filtering.go]: Done

File: cmd/rpcdaemon/commands/trace_filtering.go (hunk 2 above)
Comment (tjayrush) [reply cmd/rpcdaemon/commands/trace_filtering.go:74]: Done

**Pull request merged**